projects/demo/src/modules/components/table-pagination/examples/import/import.md

```diff
@@ -0,0 +1,13 @@
+```ts
+import {TuiTablePaginationComponent} from '@taiga-ui/addon-table';
+
+// ...
+
+@NgModule({
+  imports: [
+    // ...
+    TuiTablePaginationComponent,
+  ],
+})
+export class YourModule {}
```

Comment:
nit: in other docs we use `MyModule` or `MyComponent`, not `Your` 😄

Reply:
fixed